src/beamlime/services/dashboard.py

```diff
@@ -222,6 +247,24 @@ def update_plots(self, n: int | None):
                 fig.data[0].x = data.coords[data.dim].values
                 fig.data[0].y = data.values
 
+            for msg in detector_messages:
+                key = msg.key.source_name
+                data = msg.value
+                for dim in data.dims:
+                    if dim not in data.coords:
+                        data.coords[dim] = sc.arange(dim, data.sizes[dim], unit=None)
+                if key not in self._detector_plots:
+                    self._detector_plots[key] = self.create_detector_plot(key, data)
+                fig = self._detector_plots[key]
+                if len(data.dims) == 1:
+                    fig.data[0].x = data.coords[data.dim].values
+                    fig.data[0].y = data.values
+                else:  # 2D
+                    x_dim, y_dim = data.dims
```

Comment:
In plopp, the order is usually `y_dim, x_dim`. I don't know if it matters here though...

Reply:
Good catch, was indeed wrong!